contracts/stellar/context-config/deploy_devnet.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+set -e
+
+# Get the directory where the script is located
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Check if stellar CLI is already installed
+if ! command -v stellar &> /dev/null; then
+    echo "Installing stellar CLI..."
+    
+    # Detect OS and architecture
+    OS=$(uname -s)
+    ARCH=$(uname -m)
+    
+    # Map OS and architecture to release file names
+    case "$OS-$ARCH" in
+        "Darwin-arm64"|"Darwin-aarch64")
+            BINARY_NAME="stellar-cli-22.2.0-aarch64-apple-darwin.tar.gz"
+            ;;
+        "Darwin-x86_64")
+            BINARY_NAME="stellar-cli-22.2.0-x86_64-apple-darwin.tar.gz"
+            ;;
+        "Linux-x86_64")
+            BINARY_NAME="stellar-cli-22.2.0-x86_64-unknown-linux-gnu.tar.gz"
+            ;;
+        "Linux-aarch64"|"Linux-arm64")
+            BINARY_NAME="stellar-cli-22.2.0-aarch64-unknown-linux-gnu.tar.gz"
+            ;;
+        *)
+            echo "Unsupported platform: $OS-$ARCH"
+            exit 1
+            ;;
+    esac
+    
+    echo "Downloading $BINARY_NAME..."
+    wget "https://github.com/stellar/stellar-cli/releases/download/v22.2.0/${BINARY_NAME}"
+    tar xzf "${BINARY_NAME}"
+    chmod +x stellar
+    sudo mv stellar /usr/local/bin/
```

Comment:
we shouldn't require elevated permissions for this. Use `$HOME/.local/bin` instead?